public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js

```diff
@@ -552,6 +552,21 @@ function Onboarding(svl, actionStack, audioEffect, compass, form, handAnimation,
             modalMission.setMissionMessage(mission, neighborhood, null, function () {
                 svl.initialMissionInstruction.start(neighborhood);
             });
+            var url = '/isTurker';
+            $.ajax({
+                async: true,
+                url: url,//endpoint that checks above conditions
+                type: 'get',
+                success: function(data){
+                    if(data.isTurker){
+                        svl.ui.status.currentMissionReward.show();
```

Comment:
What is this code for? Should the reward trackers on the sidebar be visible during onboarding? It currently isn't.